prisma/migrations/20230103180746_multi_asset_rename/migration.sql

```diff
@@ -0,0 +1,97 @@
+/*
+  Warnings:
+
+  - The values [MASP_TRANSFER,MASP_BURN,MASP_MINT] on the enum `event_type` will be removed. If these variants are still used in the database, this will fail.
+  - You are about to drop the column `masp_transaction_id` on the `events` table. All the data in the column will be lost.
+  - You are about to drop the column `masp_burn_count` on the `user_points` table. All the data in the column will be lost.
+  - You are about to drop the column `masp_burn_last_occurred_at` on the `user_points` table. All the data in the column will be lost.
+  - You are about to drop the column `masp_burn_points` on the `user_points` table. All the data in the column will be lost.
+  - You are about to drop the column `masp_mint_count` on the `user_points` table. All the data in the column will be lost.
+  - You are about to drop the column `masp_mint_last_occurred_at` on the `user_points` table. All the data in the column will be lost.
+  - You are about to drop the column `masp_mint_points` on the `user_points` table. All the data in the column will be lost.
+  - You are about to drop the column `masp_transfer_count` on the `user_points` table. All the data in the column will be lost.
+  - You are about to drop the column `masp_transfer_last_occurred_at` on the `user_points` table. All the data in the column will be lost.
+  - You are about to drop the column `masp_transfer_points` on the `user_points` table. All the data in the column will be lost.
+  - You are about to drop the `masp_transaction_head` table. If the table is not empty, all the data it contains will be lost.
+  - You are about to drop the `masp_transactions` table. If the table is not empty, all the data it contains will be lost.
+  - A unique constraint covering the columns `[multi_asset_id]` on the table `events` will be added. If there are existing duplicate values, this will fail.
+
+*/
+-- AlterTable
+ALTER TABLE "events" DROP COLUMN "masp_transaction_id",
+ADD COLUMN     "multi_asset_id" INTEGER;
+
```

Comment:
Had to move these ALTER statements to top from bottom, was getting a column not found error